src/site/content/en/metrics/cls/index.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: post
+title: Cumulative Layout Shift (CLS)
+authors:
+  - philipwalton
+  - mihajlija
+date: 2019-06-11
+updated: 2019-11-07
+description: |
+  This post introduces the Cumulative Layout Shift (CLS) metric and explains
+  how to measure it
+tags:
+  - performance
+  - metrics
+---
+
+{% Aside %}
+  Cumulative Layout Shift (CLS) is an important, user-centric metric for
+  measuring [visual stability](/user-centric-metrics/#types-of-metrics) because it helps quantify how often users
+  experience unexpected layout shifts&mdash;a low CLS helps ensure that the
+  page is [delightful](/user-centric-metric/#user-centric-metric-questions).
+{% endAside %}
+
+Have you ever been reading an article online when something suddenly changes on
+the page? Without warning, the text moves, and you've lost your place. Or even
+worse: you're about to tap a link or a button, but in the instant before your
+finger lands&mdash;BOOM&mdash;the link moves, and you end up clicking something
+else!
+
+Most of the time these kinds of experiences are just annoying, but in some
+cases, they can cause real damage.
+
+<figure class="w-figure w-figure--center">
+  <video autoplay controls loop muted
+    class="w-screenshot"
+    poster="https://storage.googleapis.com/web-dev-assets/layout-instability-api/layout-instability-poster.png"
+    width="658" height="510">
+    <source
+      src="https://storage.googleapis.com/web-dev-assets/layout-instability-api/layout-instability2.webm"
+      type="video/webm; codecs=vp8">
+    <source
+      src="https://storage.googleapis.com/web-dev-assets/layout-instability-api/layout-instability2.mp4"
+      type="video/mp4; codecs=h264">
+  </video>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    A screencast illustrating how layout instability can negatively affect
+    users.
+  </figcaption>
+</figure>
+
+Unexpected movement of page content usually happens because resources are loaded
+asynchronously or DOM elements get dynamically added to the page above existing
+content. The culprit might be an image or video with unknown dimensions, a font
+that renders larger or smaller than its fallback, or a third-party ad or widget
+that dynamically resizes itself.
+
+What makes this issue even more problematic is that how a site functions in
+development is often quite different from how users experience it. Personalized
+or third-party content often doesn't behave the same in development as it does
+in production, test images are often already in the developer's browser cache,
+and API calls that run locally are often so fast that the delay isn't
+noticeable.
+
+The Cumulative Layout Shift (CLS) metric helps you address this problem by
+measuring how often it's occurring for real users.
+
+## What is CLS?
+
+CLS measures the sum of the individual _layout shift scores_ for each _expected_
+_layout shift_ that occurs between [First Contentful Paint (FCP)](/fcp/) and
```

Comment:
Small correction here - CLS doesn't wait for FCP to start tracking layout shifts.  It can report layout shifts that happen before FCP too.

Reply:
Thanks, updated.